src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -44,37 +44,44 @@
 
         <v-col
           cols="12"
-          sm="7"
+          sm="9"
+          class="d-flex justify-space-between"
         >
           <v-text-field
-            class="incorporation-number"
+            class="expro-registration-number"
             filled
             hide-details
-            label="B.C. Extraprovincial Incorporation Number"
+            label="B.C. Extraprovincial Registration Number"
             readonly
+            xclearable
```

Comment:
Is the x here a typo?

Reply:
Haha, sort of. Good catch. I was trying something, and this is my way to disable a prop temporarily. I'll remove this in my next commit.

Yes, there will be some more commits -- I'm trying to get file downloads from Minio working.

Reply:
I see, all right! Thank you, resolving.